dockertest/subtest.py

```diff
@@ -429,13 +458,62 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
+            # Ignore known problems in specific docker builds
+            if self.is_known_failure(name):
```

Comment:
Good idea, so done. (There's already a warning issued in `is_known_failure()` but this removes ambiguity, and I'm OK with having two contiguous warnings. Thanks.